p22/tests/integration_tests.rs

```diff
@@ -0,0 +1,20 @@
+#[test]
+fn tests_celsius2farenheit2celsius() {
+    let celsius: i32 = 40;
+    let farenheit: i32 = 104;
+    assert_eq!(
+        farenheit,
+        p22::calc::celsius2farenheit(p22::calc::farenheit2celsius(farenheit))
```

Comment:
It's better to import functions like `use p22::calc::{celsius2farenheit, farenheit2celsius};` in the beginning of this file to save a bit on typing and make tests more readable.